From b8950472a28c2647b6867cc002759777c5ba077a Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Wed, 10 Aug 2022 10:46:56 +0530
Subject: [PATCH] Power Optimization.

Disable home screen search
Original owner of this patch-:
Kaushlendra Kumar <kaushlendra.kumar@intel.com>

Tracked-On:
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 src/com/android/launcher3/config/FeatureFlags.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/launcher3/config/FeatureFlags.java b/src/com/android/launcher3/config/FeatureFlags.java
index e0769dbfb5..0ba9dfae0b 100644
--- a/src/com/android/launcher3/config/FeatureFlags.java
+++ b/src/com/android/launcher3/config/FeatureFlags.java
@@ -52,7 +52,7 @@ public final class FeatureFlags {
      * Enable moving the QSB on the 0th screen of the workspace. This is not a configuration feature
      * and should be modified at a project level.
      */
-    public static final boolean QSB_ON_FIRST_SCREEN = true;
+    public static final boolean QSB_ON_FIRST_SCREEN = false;
 
     /**
      * Feature flag to handle define config changes dynamically instead of killing the process.
-- 
2.17.1

